From 9276a11364668020424fb58dd71c0b9bb0b95993 Mon Sep 17 00:00:00 2001
From: Zhang Qiao <zhangqiao22@huawei.com>
Date: Sun, 24 Apr 2022 11:30:01 +0800
Subject: [PATCH 2935/4585] [Backport] sysctl: Modify
 sysctl_overload_detect_period minimum to 100ms

hulk inclusion
category: features
bugzilla: https://gitee.com/openeuler/kernel/issues/I4UQ08

Reference: https://gitee.com/openeuler/kernel/commit/7339ca6ee849697b0ced717dff8cd15543ac4b0d

--------------------------------

The sysctl_overload_detect_period indicates the maximum time that an
offline task can be preempted by online tasks. Currently, this minimum
is 1s, it is too long for the vcpu thread in VM. So set its minimum to
100ms.

Signed-off-by: Zhang Qiao <zhangqiao22@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>

Signed-off-by: Wang Hai <wanghai38@huawei.com>
---
 kernel/sysctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/sysctl.c b/kernel/sysctl.c
index d9fb660465fd..61284f25385f 100644
--- a/kernel/sysctl.c
+++ b/kernel/sysctl.c
@@ -2706,7 +2706,7 @@ static struct ctl_table kern_table[] = {
 		.maxlen		= sizeof(unsigned int),
 		.mode		= 0644,
 		.proc_handler	= proc_dointvec_minmax,
-		.extra1		= &one_thousand,
+		.extra1		= &one_hundred,
 		.extra2		= &hundred_thousand,
 	},
 	{
-- 
2.17.1

